bin/hdfs-mesos-datanode

```diff
@@ -2,24 +2,5 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-if [ -z "$JAVA_HOME" ]; then
-  JAVA_HOME_DIR=$(dirname $(readlink -f $(which java)))
-  if [ -f $JAVA_HOME_DIR/../../bin/java ]; then
-    export JAVA_HOME=$JAVA_HOME_DIR/../..
-  elif [ -f $JAVA_HOME_DIR/../bin/java ]; then
-    export JAVA_HOME=$JAVA_HOME_DIR/..
-  else 
-    echo “Error: Could not determine JAVA_HOME”
-    exit 1;
-  fi
-fi
-
-trap "{ $DIR/mesos-killtree "$$" ; exit 0; }" EXIT
-
-function run_datanode() {
-  while [ true ] ; do
-    $DIR/hdfs datanode
-  done
-}
-
-run_datanode
+$DIR/hdfs-mesos-env
```

Comment:
We need to loop through and run the datanode here in order to try to recover from failures. What if there is a network partition or a failure communicating with a NameNode or JournalNode? We don't want the task to die immediately, but we want to spend some time relaunching the task. This is why these scripts were written in the form of loops. (cc @brndnmtthws). I would also be okay writing java code that loops through and tries to launch a number of times. As another note, once we add health checks to our tasks, we will be able to kill a task if we aren't able to hit the http endpoint for datanode, namenode, etc